recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: 288509d28ed062ddf8b08212f65afb3aa07762a9a84bc5da9f7132016c8dd31f
 
 build:
-  number: 2
+  number: 3
```

Comment:
FYI with `openblas` this should start at `200` and go up from there or be reset to `200` when making a new release.

Reply:
@jakirkham OK. So should I maintain multiple builds/branches with and without openblas?

Reply:
@jakirkham should I update the build number now or wait until the next release?